windows/scripts/BinplaceCudaDependencies.cmd

```diff
@@ -0,0 +1,22 @@
+set CUDA_TOOLKIT_BIN_DIR=%~1%
+set CUDNN_PATH=%~2%
+set IS_CPU_ONLY_BUILD=%3%
+set USE_CUDNN=%4%
+set OUTPUT_DIR=%~5%
+
+if %IS_CPU_ONLY_BUILD% == true (
+    echo BinplaceCudaDependencies : CPU only build, don't copy cuda dependencies.
+ ) else (
+    echo BinplaceCudaDependencies : Copy cudart*.dll, cublas*dll, curand*.dll to output.
```

Comment:
@pavlejosipovic Why do you need to copy CUDA files to `OUTPUT_DIR`?

Also, the `CUDNN` logic doesn't apply to my case (and many others). Typically, people install cuDNN by `cp -r cuda\* $(CUDA_PATH)`. Caffe Linux build, TensorFlow, etc. assume this convention.